src/mmw/js/src/core/layerPicker.js

```diff
@@ -229,37 +230,48 @@ var LayerPickerNavView = Marionette.ItemView.extend({
         'toggle:layer': 'render'
     },
 
-
     events: {
         'click @ui.tabIcons': 'onTabIconClicked',
     },
 
-
     templateHelpers: function() {
+        var doesLayerGroupHaveActiveLayer = function(layerGroup) {
+            var layers = layerGroup.get('layers');
+            if (layers) {
+                return layers.findWhere({ active : true });
+            }
```

Comment:
Consider returning false at the end of the function just so the result is consistently a boolean.